src/core/services/theming/theming.js

```diff
@@ -145,7 +145,7 @@ var DARK_FOREGROUND = {
 };
 var LIGHT_FOREGROUND = {
   name: 'light',
-  '1': 'rgba(255,255,255,1.0)',
+  '1': 'rgb(255,255,255)',
```

Comment:
Why using `RGB` here? It's actually the same (I know), but it would keep the consistency.